build_tools/packaging/python/templates/rocm/src/rocm_sdk/__init__.py

```diff
@@ -34,10 +34,12 @@ def find_libraries(*shortnames: str) -> list[Path]:
             lib_entry = _dist_info.ALL_LIBRARIES[shortname]
         except KeyError:
             raise ModuleNotFoundError(f"Unknown rocm library '{shortname}'")
-        if not lib_entry.dllname:
+
+        if is_windows and not lib_entry.dllname:
```

Comment:
Oops, almost merged with a bug. We need to start testing these packages on PRs, not just with nightly jobs.